wpilibc/src/main/native/include/frc/Timer.h

```diff
@@ -90,16 +90,6 @@ class Timer {
    */
   bool HasElapsed(units::second_t period) const;
 
-  /**
-   * Check if the period specified has passed and if it has, advance the start
-   * time by that period. This is useful to decide if it's time to do periodic
-   * work without drifting later by the time it took to get around to checking.
-   *
-   * @param period The period to check for.
-   * @return       True if the period has passed.
-   */
-  bool HasPeriodPassed(units::second_t period);
-
```

Comment:
Did we forget to deprecate this before?

Reply:
Yes.

Reply:
Looking back in history, It was proposed to be deprecated mid-season 2020, but we didn't want to deprecate something mid-season. It probably should have been deprecated before 2021.

Reply:
Let's deprecate it instead of removing it.

Reply:
Done: https://github.com/wpilibsuite/allwpilib/pull/3598